app/reducer.js

```diff
@@ -1,3 +1,11 @@
+import { createReducer } from '@reduxjs/toolkit'
+import { setAutoFreeze } from 'immer'
```

Comment:
redux-toolkit internally uses immer. didnt seprately install it to avoid version conflict issues.